src/main/java/com/productdock/library/inventory/application/port/in/UpdateBookStatusUseCase.java

```diff
@@ -0,0 +1,8 @@
+package com.productdock.library.inventory.application.port.in;
+
+import com.productdock.library.inventory.domain.RentalRecord;
+
+public interface UpdateBookStatusUseCase {
```

Comment:
Maybe it's better to name it `UpdateBookStateUseCase`

Reply:
Considering we already use "Status" wording in rentalStatus, similar should be applied here